specification/securitycenter/resource-manager/readme.md

```diff
@@ -0,0 +1,59 @@
+# Security Center
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Security.
+
+
+
+---
+## Getting Started 
+To build the SDK for Security, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information 
+These are the global settings for the Security API.
+
+``` yaml
+openapi-type: arm
+tag: package-composite-v1
```

Comment:
Tags should usually correspond to the api version which is `2017-08-01-preview` in this case